lit/strings_holder.py

```diff
@@ -2,68 +2,87 @@
+                PATH_1_NAME = 'first'
+                PATH_1_HELP = 'path to first file'
+                PATH_2_NAME = 'second'
+                PATH_2_HELP = 'path to second file'
+
         class Init(Enum):
-            NAME = 'init';
-            HELP = 'initializes repository in the current directory';
+            NAME = 'init'
+            HELP = 'initializes repository in the current directory'
+
         class Log(Enum):
-            NAME = 'log';
-            HELP = 'shows lit log';
+            NAME = 'log'
+            HELP = 'shows lit log'
+
         class Rm(Enum):
-            NAME = 'rm';
-            HELP = 'removes files from staging area';
+            NAME = 'rm'
+            HELP = 'removes files from staging area'
+
             class Arguments(Enum):
-                PATH_NAME = 'path';
-                PATH_HELP = 'path to file';
+                PATH_NAME = 'path'
+                PATH_HELP = 'path to file'
+
         class Status(Enum):
-            NAME = 'status';
-            HELP = 'shows repository state';
-    class Settings(Enum):
-        TRACKED_FILE_PATH = '.lit/tracked_files.json';
-        COMMIT_LOG_PATH = '.lit/commits_log.json';
-        class Log(Enum):
-            COMMIT = 'Commit: ';
-            COMMIT_MESSAGE = 'Commit message: ';
-            USERNAME = 'Username: ';
-            DATE = 'Date: ';
-        class Init(Enum):
-            LIT = '.lit';
-            LIT_INITED = 'LIT has been already inited in this directory';
-            COMMIT_DIR = 'commits';
-            TRACKED_FILE = 'tracked_files.json';
-            COMMIT_LOG = 'commits_log.json';
-            TRACKED_FILE_INIT = '{"files": []}';
-            COMMIT_LOG_INIT = '{"commits":[]}';
-        class Commit(Enum):
-            FILES_IN_COMMIT_DIR = '.lit/commits/';
-            ZIP_FILE_NAME = '.lit/commits/hash';
-            ZIP_EXTENCION = '.zip';
-            USER = 'user';
-            LONG_HASH = 'long_hash';
-            SHORT_HASH = 'short_hash';
-            DATETIME = 'datetime';
-            COMMENT = 'comment';
+            NAME = 'status'
+            HELP = 'shows repository state'
+
+    class InitSettings(Enum):
+        LIT_DIR = '.lit'
+        LIT_INITED = 'LIT has been already inited in this directory'
+
+    class TrackedFileSettings(Enum):
+        PATH = '.lit/tracked_files.json'
+        FILE_NAME = 'tracked_files.json'
+        FILES_KEY = 'files'
+        INIT_CONTENT = '{"files": []}'
+
+    class LogSettings(Enum):
+        COMMIT = 'Commit: '
+        COMMIT_MESSAGE = 'Commit message: '
+        USERNAME = 'Username: '
+        DATE = 'Date: '
+        PATH = '.lit/commits_log.json'
+        FILE_NAME = 'commits_log.json'
+        INIT_CONTENT = '{"commits":[]}'
+        KEY = "commits"
+
+    class CommitSettings(Enum):
+        DIR_PATH = '.lit/commits/'
+        DIR_NAME = 'commits'
+        ZIP_FILE_NAME = '.lit/commits/hash'
+        ZIP_EXTENSION = '.zip'
+        USER = 'user'
```

Comment:
User name should be changeable externally from CLI. Here we can add some method to get/set user name and store it on disk in JSON/SQLite/etc.

Reply:
Probably best way to store settings it's using configuration files. Check configparser module.

Reply:
This settings are constants, they will not be changed. Configparse good idea for implementing storing user information. I other task.